tools/operator/testdata/sbecc-test-input.yml

```diff
@@ -0,0 +1,98 @@
+organizations:
+  - name: org1
+    connProfilePath: ./connection-profile
+  - name: org2
+    connProfilePath: ./connection-profile
+  - name: org3
+    connProfilePath: ./connection-profile
+  - name: org4
+    connProfilePath: ./connection-profile
```

Comment:
The other test-input files contain a full path name.
./connection-profile/connection_profile_org1.yaml
 Is there a reason for that, whereas this one contains only ./connection-profile ?

Reply:
For sbe, we are using List in targetPeers. In that case, we will be having peers from different orgs.  If we pass the whole connection profile file path instead of a directory, the pte will not be able to read the connection profile of other org. For example, if we have
 listOpt:
      org1:
        - peer0-org1
      org2:
        - peer0-org2
and we pass the connection profile file path of org1, then pte will not be able to find the connection profile for org2.

Reply:
ok. right.